virttest/utils_test/libvirt_ceph_utils.py

```diff
@@ -0,0 +1,152 @@
+"""
+High-level libvirt ceph utility functions.
+
+This module is meant to reduce code size by performing common ceph setup procedures.
+:copyright: 2021 Red Hat Inc.
+"""
+
+import os
+import logging
+
+from avocado.utils import process
+
+from virttest import data_dir
+from virttest import virsh
+from virttest import utils_package
+from virttest import ceph
+
+from virttest.utils_test import libvirt
+
+from virttest.libvirt_xml import vm_xml
+from virttest.libvirt_xml.devices.disk import Disk
+
+
+def create_ceph_backend_vm_disk(vm, params, is_setup=True):
```

Comment:
We'd better build a new function for disk removal. How do you think?

Reply:
The logic and pattern  are very similar to existed utils such as :libvirt.setup_or_cleanup_iscsi,setup_or_cleanup_gluster.
change name to create_or_cleanup_ceph_backend_vm_disk

Reply:
Yes, that's right, but that shouldn't be best practice. We'd better to ask a function to do one thing to make the code clean.

Reply:
@kylazhang 
@dzhengfy 

Do we have guidelines that this is not best practices?
Since if we go through utils module, most of functions or methods achieve  more than one single function, otherwise we may have much more numbers of functions or methods maintained? I assume we need keep balance among them

Reply:
I agree with @Yingshun , the old way might not be the best. Splitting the function to 2 would make the calling of functions much more clear.

Reply:
Regarding API exposed by module, we need look at them from different levels(low level and high level).
We position utilities as high level. which provides quite complex or complicated functionality.
If someone can look at tensorflow or multiprocessing in python, both of them provides very simple API with very high level abstraction to users, but actually implement quite complicated logiic  in underlying logic. 
Without high level abstraction, we hardly imagin that we can achieve machine leaning in just a few lines. Or implement parellel handliing by calling  a few lines code of multiple processing

Reply:
environment setup and clean up are different operations and will be executed in different stages. We usually call a function to setup it and call a function to clean up it, no matter how big the project is.
I still think we should split them, but it's up to you.